docs/content/commands/update.md

```diff
@@ -65,3 +65,10 @@ If you want to update a single group you can use the following command:
   `--redirects`: Creates binding redirects for the NuGet packages.
 
   `--no-install`: Skips paket install --hard process afterward generation of [`paket.lock` file](lock-file.html).
+
+## Updating http dependencies
+
+The update command has no effect on http dependencies. If you want to update a file you need to use the [`paket install` command](paket-install.html) with `--force [-f]` option.
```

Comment:
If that's true then this is clearly a bug. @sergey-tihon is that correct? I mean you use https://github.com/sergey-tihon/Stanford.NLP.NET/blob/master/paket.dependencies#L13